packages/trader/src/App/Components/Elements/PositionsDrawer/result-mobile.tsx

```diff
@@ -1,11 +1,15 @@
 import classNames from 'classnames';
-import PropTypes from 'prop-types';
 import React from 'react';
 import { CSSTransition } from 'react-transition-group';
 import { Icon } from '@deriv/components';
 import { localize } from '@deriv/translations';
 
-const ResultMobile = ({ is_visible, result }) => {
```

Comment:
I know it is out of the scope of the card but can you help to rename this file to a more descriptive name 🙏🏼

Reply:
👍